Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.js

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (C) 2023 Apple Inc. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS ``AS IS''
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+ * THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS
+ * BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+ * THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+WI.FontVariationDetailsSectionRow = class FontVariationDetailsSectionRow extends WI.DetailsSectionRow
+{
+    constructor(fontVariationAxis, abortSignal)
+    {
+        super();
+
+        this.element.classList.add("font-variation");
+
+        let {name, tag, value, defaultValue, minimumValue, maximumValue} = fontVariationAxis;
+
+        console.assert(typeof tag === "string" && tag.length === 4, "Invalid font variation axis tag", tag);
+
+        this._labelElement = document.createElement("div");
+        this._labelElement.className = "label";
+        this.element.appendChild(this._labelElement);
+
+        this._tagElement = document.createElement("div");
+        this._tagElement.className = "tag";
+        this.element.appendChild(this._tagElement);
+
+        this._variationRangeElement = document.createElement("div");
+        this._variationRangeElement.className = "variation-range";
+        this.element.appendChild(this._variationRangeElement);
+
+        this._inputRangeElement = document.createElement("input");
+        this._inputRangeElement.setAttribute("type", "range");
+        this._inputRangeElement.setAttribute("name", tag);
+        this._inputRangeElement.setAttribute("min", minimumValue);
+        this._inputRangeElement.setAttribute("max", maximumValue);
+        this._inputRangeElement.setAttribute("value", value ?? defaultValue);
+        this._inputRangeElement.setAttribute("step", this._getAxisResolution(minimumValue, maximumValue));
```

Comment:
All of these attributes have native JavaScript members we can use on the element. e.g.
```
this._inputRangeElement.type = "range";
```

Reply:
Updated to properties on the object.